PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/WindowsAzurePreferencePage.java

```diff
@@ -133,18 +134,20 @@ public boolean performOk() {
 					String oldPrefVal = DataOperations.getProperty(dataFile, Messages.prefVal);
 					DataOperations.updatePropertyValue(doc, Messages.prefVal,
 							String.valueOf(btnPreference.getSelection()));
-					String version = DataOperations.getProperty(dataFile, Messages.version);
+					
+					final String version = DataOperations.getProperty(dataFile, Messages.version);
+					final String newVersion = Activator.getDefault().getBundle().getVersion().toString(); 
```

Comment:
Both `getBundle()` and `getVersion` won't be `null` according http://www.docjar.com/html/api/org/eclipse/core/runtime/Plugin.java.html and https://osgi.org/javadoc/r5/core/org/osgi/framework/Bundle.html#getVersion().